src/components/ColorInputFlyout/ColorPickerFlyout.stories.tsx

```diff
@@ -14,26 +14,41 @@ export default {
         disabled: false,
         currentColor: null,
         clearable: false,
+        showDelete: false,
```

Comment:
Only the story example is missing an onDelete method, mainly because Prettier was giving me errors due to it being unused in the story example, but I can add it back in.